src/main/java/com/github/manolo8/darkbot/config/actions/values/StatTypeValue.java

```diff
@@ -35,11 +39,18 @@ public class StatTypeValue implements Value<Number>, Parser {
         }
     }
 
-    private Stats.General getKeyFromString(String key) {
-        for (Stats.General stat : Stats.General.values()) {
-            if (stat.name().equalsIgnoreCase(key)) return stat;
-        }
-        return null;
+    private StatsAPI.Key getKeyFromString(String token) {
+        String[] tokenParts = token.split(":");
```

Comment:
```suggestion
        String[] tokenParts = token.split(":", 3);
```

Reply:
If you don't cap it at 3, you may have a string like:
`plugin:something:general:uridium`
which would be parsed as
namespace=plugin, category=general, name=uridium, leaving the "something" being completely ignored. This is faulty behavior, so you either limit it to 3 splits or you throw an error if there's more than 3, but it needs to be handled